lightning/models/cookie.py

```diff
@@ -0,0 +1,38 @@
+from lightning_adapter.models import Cookie as Cookie_gen
+from ..exceptions import CookieException
+
+
+class Cookie(dict):
+    attribute_map = {
```

Comment:
Instead of this map, should just require only keyword parameters explicitly. https://medium.com/analytics-vidhya/keyword-only-arguments-in-python-3c1c00051720

```
def __init__(self, *, name, value, path=None, domain=None, ...):
```

Reply:
refactored in 8c543aa7094c15a53e877f4037bc152f2c1099b3